.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Description
+<!-- Provide a summary of what the PR does and why it is being submitted. -->
+
+## Testing
+<!-- Describe how the changes have been tested. Provide test instructions or details. -->
+
+## External links
+<!-- Any related external links can be added here -->
+
+- [JIRA](https://hashicorp.atlassian.net/browse/xxxx)
+- [RFC](https://docs.google.com/document/d/xxxx)
+- [PRD](https://docs.google.com/document/d/xxxx)
+- [Related PR](https://github.com/hashicorp/atlas/pull/xxxx)
+- [Documentation](https://github.com/hashicorp/terraform-docs-common/pull/xxxx)
```

Comment:
Is there a reason for these link examples?

This is a public repository, so external contributors won't be using Jira, RFCs, or PRDs and it's likely internal contributors won't have 2 of these 3 links depending on the PR request purpose. The bottom two are "randomly selected" pointers, one of which is not a public repo. I am of the opinion these should be removed.